db/migrations/003_teams.sql

```diff
@@ -10,11 +10,28 @@ CREATE TABLE team (
   source TEXT NULL,
   created_by UUID NOT NULL,
   created_at timestamp NOT NULL DEFAULT now(),
-  updated_at timestamp NOT NULL DEFAULT now(),
-  FOREIGN KEY (created_by) REFERENCES person(id)
+  updated_at timestamp NOT NULL DEFAULT now()
 );
 
-ALTER TABLE person ADD  FOREIGN KEY (team_id) REFERENCES team(id);
+CREATE TABLE person (
+  id UUID DEFAULT generate_ulid() PRIMARY KEY,
+  name TEXT NOT NULL,
+  avatar TEXT NULL,
+  team_id UUID NULL, -- every team is also a person
```

Comment:
So, teams can have multiple people. But every person is associated with at least one team?